src/components/StateCard/RefreshStateCard.tsx

```diff
@@ -1,13 +1,20 @@
+import { Spinner } from '@patternfly/react-core';
+
 import styles from './RefreshStateCard.module.css';
 
-interface IRefreshStateCard {}
+interface IRefreshStateCard {
+  isInline?: boolean;
+}
 
 /**
  * Refresh State component with loading indicator displayed over real data that were valid for previous state.
  */
-export const RefreshStateCard = ({ children }: React.PropsWithChildren<IRefreshStateCard>) => (
-  <div className={styles['refresh-state-card']}>
-    <div className={styles['refresh-state-card__body']}>{children}</div>
-    <div className={styles['refresh-state-card__indicator']}></div>
-  </div>
-);
+export const RefreshStateCard = ({ children, isInline = false }: React.PropsWithChildren<IRefreshStateCard>) =>
+  isInline ? (
+    <Spinner isInline isSVG aria-label="Loading..." />
```

Comment:
There are already 3 places where Spinner is used (search `<Spinner`).

Let's create `component/Spinner/Spinner.tsx` and reuse it everywhere. Import PatternFly  spinner internally as `SpinnerPF`.